lib/winston/transports/http.js

```diff
@@ -35,6 +35,7 @@ module.exports = class Http extends TransportStream {
     this.port = options.port;
     this.auth = options.auth;
     this.path = options.path || '';
+    this.maximumDepth = options.maximumDepth || 10;
```

Comment:
We can't change the current default behavior though -- looks like default is `Infinity` per https://github.com/BridgeAR/safe-stable-stringify/blob/46f24495221803b7a945883cb8fb262b14f1ea52/index.js#L125 so I think that is what we should use here.

Reply:
(Can't quite use Infinity due to the checks `safe-stable-stringify` has in place -- try using ES6 spreading to conditionally set the prop in the config object if it's passed in as an option)